components/match2/wikis/rocketleague/match_summary.lua

```diff
@@ -154,48 +197,188 @@ function CustomMatchSummary.getByMatchId(args)
 		end
 	end
 
-	local footerSet = false
-	local footer = mw.html.create('div')
-		:addClass('brkts-popup-footer')
-	local footerSpacer = mw.html.create('div')
-		:addClass('brkts-popup-spaced')
-	if not Logic.isEmpty(match.extradata.octane) then
-		footerSet = true
-		footerSpacer:node('[[File:Octane_gg.png|14x14px|link=http://octane.gg/matches/' ..
-			match.extradata.octane ..
-			'|Octane matchpage]]')
+	if
+		Table.isNotEmpty(vods) or
+		String.isNotEmpty(match.vod) or
+		not Logic.isEmpty(match.extradata.octane)
```

Comment:
Seems to be a few issues with Octane. Let's make an issue on it.
(using extradata instead of links, not storing full url)

Reply:
#1141